### ISSUE TITLE
JECC - Analysis - X-axis ist not aligned between different diagramms

### ISSUE THREAD

**Issue by EnvidatecNH** [OpenJEVis/JEVis#2347]
X-axis ist not aligned between different diagramms

![grafik](https://user-images.githubusercontent.com/6728123/147773571-a52310f4-40d9-4550-985f-7d5105ffcd47.png)

-> find the analysis on FuE-Server -> Envidatec EN-4

**Comment by gschutz84** [comment]
this is because the second chart has a second y-axis

**Comment by EnvidatecNH** [comment]
That's the cause not a solution. The same problem appears if the y-axis values have different lengths. In case of 1 chart has a second axis, all other charts needs empty y-axis on the right with sam width or sth comparable.

**Comment by gschutz84** [comment]
like 2 years ago, there is no solution yet. its not possible to make a "invisible" y-axis. even if the units have a different letter height, e.g. m and kWh, the axes wont be aligned properly. also the java 8 lib isnt developed anymore, only java 11 is continued unfortunately

**Comment by EnvidatecNH** [comment]
Is it possible to set the the label font color for the right y-axis separately?

**Comment by gschutz84** [comment]
no, the css funcionalities are very limited. the main problem is, a axis needs data w/o it wont be displayed, its somewhere deep in the chart lib code

**Comment by EnvidatecNH** [comment]
If 1 chart has 2 axis and other have just 1, it could be a (very dirty) workarround to draw the charts with 1 axis on both. The data are already loaded and the line will be the same -> TBD
